tests/virtctl/usbredir.go

```diff
@@ -194,7 +194,7 @@ func runConnectGoroutine(cmd *cobra.Command, errch chan error) {
 		// Ends when PipeWritter closes or after find ip.
 	}(rOut)
 
-	// Make local bufferred to not block select
+	// Make remote buffer non-blocking for select
```

Comment:
Now the problem is this one :)
```suggestion
	// Make local buffer non-blocking for select
```